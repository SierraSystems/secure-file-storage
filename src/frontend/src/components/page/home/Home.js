import React from "react";
import { NavBar } from "../../base/navbar/NavBar";
import Heading from "../../base/heading/Heading";
import { Login } from "../../composite/login/Login";

export default function Home() {
  return (
    <>
      {/* Navigation */}
      <NavBar />

      {/* Heading */}
      <Heading />

      <br />

      <Login />
    </>
  );
}
