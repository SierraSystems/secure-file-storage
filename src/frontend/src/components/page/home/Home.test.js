import React from "react";
import Home from "./Home";
import testBasicSnapshot from "../../../TestHelper";

describe("Home", () => {
  test("Component renders as expected", () => {
    testBasicSnapshot(<Home />);
  });
});
