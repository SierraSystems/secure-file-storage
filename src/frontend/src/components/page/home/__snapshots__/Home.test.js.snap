// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component renders as expected 1`] = `
Array [
  <div
    className="nav-div"
  >
    <nav
      className="navbar navbar-expand-md"
    >
      <div
        className="collapse show navbar-collapse"
        style={Object {}}
      >
        <ul
          className="mr-auto navbar-nav"
        >
          <span
            className="navbar-text"
          >
            Secure File Upload
          </span>
        </ul>
      </div>
    </nav>
  </div>,
  <br />,
  <h2>
    NTT Data Secure File Upload
  </h2>,
  <div>
    <div
      className="App"
    >
      <div
        className="filepond--wrapper"
      >
        <input
          multiple={true}
          type="file"
        />
      </div>
    </div>
  </div>,
]
`;
